source/features/dim-bots.tsx

```diff
@@ -19,9 +20,16 @@ const botNames = [
 	'weblate',
 ] as const;
 
+// All co-authored commits are excluded because it's unlikely that any bot co-authors with another bot, but instead they're co-authored with a human. In that case we don't want to dim the commit.
 const commitSelectors = [
-	...botNames.map(bot => `.commit-author[href$="?author=${bot}"]`),
-	'.commit-author[href$="%5Bbot%5D"]', // Generic `[bot]` label in author name
+	// Co-authored commits are excluded because their avatars are not linked
+	...botNames.map(bot => `div[data-testid="author-avatar"] a[href$="?author=${bot}"]`),
+	'div[data-testid="author-avatar"] a[href$="%5Bbot%5D"]', // Generic `[bot]` label in avatar in New view
+
+	// Legacy view, still used by PR commits
+	// :only-child excludes co-authored commits
+	...botNames.map(bot => `.commit-author[href$="?author=${bot}"]:only-child`),
+	'.commit-author[href$="%5Bbot%5D"]:only-child', // Generic `[bot]` label in author name
```

Comment:
I figured we could change the selector to avoid the JS check. In this case, `:only-child` matches the new view behavior